packages/docs/src/components/content/PropTableHtmlElementRow.tsx

```diff
@@ -0,0 +1,39 @@
+import humanizeList from 'humanize-react';
+import { TableRow, TableCell } from '@cmsgov/design-system';
+
+export interface PropTableHtmlElementRowProps {
+  /**
+   * List of element names
+   */
+  elements: string[];
+}
+
+/**
+ * If the component we're documenting passes extra props through to an HTML element,
+ * we want to include documentation about that. Rather than listing out all the
+ * additional attributes for those HTML elements in our props table, we will add
+ * some explanatory text with MDN documentation links to the bottom of our props
+ * table.
+ */
+const PropsTableHtmlElementRow = ({ elements }: PropTableHtmlElementRowProps) => {
+  const elementNames = elements.map((name) => <code key={name}>{`<${name}>`}</code>);
+  const elementLinks = elements.map((name) => (
+    <a href={`https://developer.mozilla.org/en-US/docs/Web/HTML/Element/${name}`} key={name}>
+      {name}
```

Comment:
I think the names should be wrapped in brackets to denote the HTML element referenced. 

MDN page titles wrap the names of these elements in them and it looks a little odd reading the word "div" without brackets or formatted as code in our docs.

Reply:
I've gone back and forth on this. https://github.com/CMSgov/design-system/pull/2239 has a change (underlined code blocks in anchor tags) that will make this look better. As it stands, if you wrap a code tag in a link, you can't tell it's a link on our site.

Reply:
I've changed the styles for code blocks in links so they look like links (kind of like [`<github>` does](#)). I've also taken out the vertical padding of code blocks everywhere to make them render more consistently in different contexts. During the blog post work, @babsdenney suggested we take out the vertical padding when they're in paragraphs so they don't alter the line heights, which makes a paragraph look wonky. That change didn't affect the table content, so instead of adding another exception, I decided to make it consistent for all contexts. I left some horizontal padding to expand the background a bit for legibility because it doesn't affect line height. What do you think, @line47 and @babsdenney?

Here's what it looks like now:
<img width="864" alt="Screen Shot 2022-11-08 at 9 56 38 AM" src="https://user-images.githubusercontent.com/7595652/200640479-8100a452-e4ae-434b-9b7e-176b91ec851f.png">